test/src/test/java/aima/test/unit/environment/map2d/PrintRomaniaMap.java

```diff
@@ -0,0 +1,66 @@
+package aima.test.unit.environment.map2d;
```

Comment:
This is kind of neat. Would it be possible to add this as a separate standalone pull request whereby:
1. It is located in aima.core.environment.map2d
2. Is generalized to printing an Map3D and renamed to PrintMap2D
3. Include a main(String[])  and updates inner methods to support specifying where the output should be written.

If this has any dependencies on third party libraries (I don't think it does) then it should reside in the 'extra' module and have its package named updated to reflect this.